iron-selectable.html

```diff
@@ -310,11 +310,6 @@
     },
 
     _activateHandler: function(e) {
-      // TODO: remove this when https://github.com/Polymer/polymer/issues/1639 is fixed so we
```

Comment:
Hmm, I'd think you also need to actually call `unlisten` somewhere if you remove this?